docs/guide/walkthrough/goal.md

```diff
@@ -36,16 +36,28 @@ export class PingCommand extends Command {
 ```
 ### Using @sern/handler
 ```ts title="commands/ping.ts" showLineNumbers
-import { commandModule, CommandType } from '@sern/handler'
-import { publish } from '../plugins';
+import { commandModule, CommandType } from '@sern/handler';
+import { PermissionFlagsBits } from 'discord.js';
+
+export const config = {
+  dmPermission: true || false, //only choose one!
```

Comment:
i think you should replace it with "boolean", which will require the user to change it accordingly but will not show this bad and redundant code

Reply:
I done that below but forgot to place above. oops